pymatgen/core/periodic_table.py

```diff
@@ -601,20 +601,38 @@ def from_Z(z: int) -> "Element":
         raise ValueError("No element with this atomic number %s" % z)
 
     @staticmethod
-    def from_row_and_group(row: int, group: int) -> "Element":
+    def from_row_and_group(pseudo_row: int, pseudo_group: int) -> "Element":
```

Comment:
This is a breaking change. Can we keep the old argument names? Clarifying in the docstring as you have should be sufficient here. I appreciate the numbers returned are also a breaking change, but no need to introduce an API change on top of it.

Otherwise this PR looks great, thanks for addressing this!

Reply:
Definitely can revert that! Since it they were positional args, I wasn't sure that it mattered much, but you're right that it should be kept the same and just have the docstrings be clear. Will get on that.

Reply:
Positional args can also optionally be used as keyword arguments -- actually useful, if you want to be very explicit and notified of API changes (which is an argument _against_ keeping them the same actually). I sometimes do it in my own code for readability.

Reply:
Also, if you're like me, you often just forget they're optional keywords and specify them anyway sometimes...